### ISSUE TITLE
`make deploy-dev` command failed

### ISSUE THREAD

**Issue by duker33** [fidals/stroyprombeton#289]
How to reproduce:
- Drop local db
- do `make deploy-dev`
- get this error:
```
FileNotFoundError at /
[Errno 2] No such file or directory: '/usr/app/src/media/flatpage/15/1531188058.0034819.jpg'
```

seems it's about wrong fixtures

**Comment by duker33** [comment]
just regenerated fixtures after little debugging

**Comment by duker33** [comment]
seems erorr not solved yet, i dig again

**Comment by 0pdd** [comment]
@duker33 the puzzle [#296](https://github.com/fidals/stroyprombeton/issues/296) is still not solved.

**Comment by 0pdd** [comment]
@duker33 the puzzle [#384](https://github.com/fidals/stroyprombeton/issues/384) is still not solved; solved: [#296](https://github.com/fidals/stroyprombeton/issues/296).
